telcell/utils/utils.py

```diff
@@ -0,0 +1,6 @@
+from telcell.data.models import Track
+
+
+def is_colocated(track_a: Track, track_b: Track) -> bool:
+    """Checks if two tracks are colocated to each other."""
+    return track_a.owner == track_b.owner
```

Comment:
This might get confusing when a track is of an unknown origin, assuming that the owner would be `None`. Following this, two tracks of unknown origin would appear to be colocated :thinking:

Reply:
On the other hand, the `owner` field can be interpreted as an attribution which may be subject to verification. The name "owner" may be not the best name though.

Reply:
The `Track` class is not tightly coupled with the current parser we use to parse the `measurements.csv` file though -- just because one parser may fail to parse `Track`s without an owner does not mean `None` values for the `owner` should not be accepted in the more general case.